### PR TITLE
AC-5626: Support for Analysis Display

### PR DESCRIPTION
**Pull request by mc-nathan**
To test:
Once everything is running, select "2017-04 BOS CH IL Round 1" from the drop down and wait about 10 seconds for the results to show up.  Should see the data requested in the ticket as well as the data needed to drive Hugh's UI.  Note: This ticket does not attempt to implement Hugh's UI.  That is handled in AC-5803.  It should look something like the attached image.

screen shot 2018-07-11 at 4 04 06 pm

### CODE REVIEW

**Comment by jonkiparsky** [comment]
Note: took a while to figure out what "everything is running meant", more mint on the pillow would have helped!

File: src/judging_round_selector.js
Comment (jonkiparsky) [inline src/judging_round_selector.js:19]: Not a blocker, but just as with django, I wish there were better ways to build this sort of URL.

File: src/utils.js
Comment (jonkiparsky) [inline src/utils.js:3]: Also not a blocker, but at some point we're going to start hating this stuff.

**Review by jonkiparsky — APPROVED** [review]
I was a bit surprised to see no tests, but I realized that this is just placeholder code, so I'm okay with that. I see that the impact changes are tested, so that's fine.

**Comment by mc-nathan** [comment]
I agree with your comments, but don't think any of them warrant a change for this PR.  I'd rather just get this one marked as done and move on.  We'll have another bite at both tests and URL handling before this goes to the users.

**Comment by jonkiparsky** [comment]
Agreed. Holding off on this merge until the corresponding PR on impact merges

**Pull request merged**
